core/templates/dev/head/domain/statistics/LearnerActionObjectFactorySpec.js

```diff
@@ -39,9 +39,9 @@ describe('Learner Action Object Factory', function() {
       actionCustomizationArgs: {},
       schemaVersion: 1
     };
-    var learnerActionObject = (
+    var learnerActionObject =
       this.LearnerActionObjectFactory.createFromBackendDict(
-        learnerActionBackendDict));
+        learnerActionBackendDict);
```

Comment:
nit: just feed the dict directly into the function since that's the only place you're using it.

Reply:
Done.